python_modules/dagster/dagster/core/definitions/input.py

```diff
@@ -273,6 +273,18 @@ def combine_with_inferred(
             asset_partitions=self._asset_partitions_fn,
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":86,"prLink":"https://app.graphite.dev/github/pr/dagster-io/dagster/7395"}} -->
<i class='graphite__hidden'>[Re: line 86]<br/></i>

[1] update this (not sure how this renders on github directly, but its the last arg in InputDef init )<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7395?utm_source=unchanged-line-comment">Graphite</a>.</span>